### PR TITLE
Make kubernetes/contrib use the approve plugin instead of the approval-handler munger.

### PR DESCRIPTION
**Pull request by cjwagner**
/cc @cblecker

### CODE REVIEW

File: prow/plugins.yaml [hunk 1]
```diff
@@ -14,7 +14,8 @@ triggers:
 
 approve:
 - repos:
-  - kubernetes
+  - kubernetes/test-infra
+  - kubernetes/kubectl
   implicit_self_approve: true
```
Comment (cjwagner) [inline prow/plugins.yaml]: k/contrib will default to `implicit_self_approve: false` and `issue_required: false`.

**Comment by cblecker** [comment]
/cc @kubernetes/contrib-maintainers 
I don't do much with contrib

**Comment by cjwagner** [comment]
I'm going to move forward with this. It shouldn't cause any behavioral changes besides being much more responsive and fixing some bugs.

**Comment by cblecker** [comment]
SGTM

**Pull request merged**